src/main/ts/constants.ts

```diff
@@ -19,7 +19,8 @@
  */
 export const GRAPH_HEIGHT = 300;
 export const GRAPH_WIDTH = 500;
-export const GRAPH_VERTICAL_MARKER_Y_POSITION_OFFSET = 15;
+export const GRAPH_VERTICAL_LABEL_LEFT_MARGIN = 5;
```

Comment:
NP: To give it a bit more space. Feel free to ignore.
```suggestion
export const GRAPH_VERTICAL_LABEL_LEFT_MARGIN = 10;
```